databroker/broker.py

```diff
@@ -1069,6 +1085,59 @@ def export_size(self, headers):
 
         return total_size * 1e-9
 
+    def fill_events(self, events_iterable, descriptors,
+                    fields=True, inplace=False):
+        """Fill a sequence of events
+
+        """
+        if fields is True:
+            fields = None
+        elif fields is False:
+            # if no fields, we got nothing to do!
+            for ev in events_iterable:
+                yield ev
```

Comment:
Explicitly comment here that we assume we don't need prepare_hook because this event has already had that applied in the first place.